### ISSUE TITLE
General Training/Inference scripts?

### ISSUE THREAD

**Issue by jchen42703** [jchen42703/kits19-cnn#7]
It would make training a lot easier to do. I'm thinking of having the ability to specify certain parameters like:
* weights_path
* fold json path
* data_path
* batch size
* n_pos
* learning rate
* epochs
* max_queue_size

This way, to test out new experiments, you can just create a new branch and just change some of the content in the `train.py` and `inference.py`.
